backend/app/Models/Invoice.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Invoice extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'epl_id',
+        'cus_id',
+        'isOnline',
```

Comment:
```suggestion
        'is_online',
```
Thường camel_case sẽ dùng trong js, BE thường dùng snake case hơn